crates/subspace-wasm-tools/src/lib.rs

```diff
@@ -46,13 +47,32 @@ pub fn create_runtime_bundle_inclusion_file(
 
     let execution_wasm_bundle_rs_path =
         env::var("OUT_DIR").expect("Set by cargo; qed") + "/" + target_file_name;
-    let execution_wasm_bundle_rs_contents = format!(
+    let mut execution_wasm_bundle_rs_contents = format!(
         r#"
             pub const {bundle_const_name}: &[u8] = include_bytes!("{}");
         "#,
         execution_wasm_bundle_path.escape_default()
     );
 
+    if let Some(link_section_name) = maybe_link_section_name {
+        let wasm_bundle_content = std::fs::read(&execution_wasm_bundle_path).unwrap_or_else(|e| {
+            panic!("Failed to read wasm bundle from {execution_wasm_bundle_path:?}: {e}",)
+        });
+        let wasm_bundle_size = wasm_bundle_content.len();
+
+        let link_section_decl = format!(
+            r#"
+        const _: () = {{
```

Comment:
I think I saw it before, but why is `const _: () = {{` necessary here, wouldn't it work without it?